### PR TITLE
Added content

### PR DESCRIPTION
**Pull request by IrKor**
Added FAQ, Resources, Current Problems, Useful links.

### CODE REVIEW

**Review by skywinder — CHANGES_REQUESTED** [review]
Let’s move faq to wiki. Just put a link to this section in the bottom

**Comment by IrKor** [comment]
> Let’s move faq to wiki. Just put a link to this section in the bottom

Done

File: README.md [hunk 1]
```diff
@@ -3,17 +3,60 @@
...
+- [Facebook’s cryptocurrency is a centralized wolf in decentralized sheep’s clothing](https://qz.com/1649526/facebook-is-begging-us-to-trust-libra-but-should-we/) by Joe Lubin, June 21, 2019
+- [A technical perspective on Facebook’s LibraBFT Consensus algorithm](https://www.theblockcrypto.com/2019/06/19/a-technical-perspective-on-facebooks-librabft-consensus-algorithm/?fbclid=IwAR2W00G6kEg7QJQdRNC806mFrSxAZxUJHb9qa8pi1eRZEzXo1nHFz6_OgiE) by Contributor Network, June 19, 2019
+- [First Look: Libra](https://info.binance.com/en/research/marketresearch/libra.html) by Binance Research, June 18th 2019
+- [A short thread on HotStuff vs Tendermint for Libra](https://twitter.com/alexskidanov/status/1141040292767789056) by Alex Skidanov, June 18th 2019
+
+
+## Current Problems
+
+Regulators in countries where there are no clear rules for working with cryptocurrencies can stand on the way of launching Libra. In may 2019, the USA Senate Banking Committee wrote an open letter to Facebook, demanding to disclose the details of the project. Senators are interested in how the company will process users' personal data.
```
Comment (skywinder) [inline README.md]: What about sources (links) to this statements?

Comment (IrKor) [reply README.md]: Links to sources are in the FAQ (wiki), so I moved this section to the wiki.

**Pull request merged**